sentry/table_sentry_project.go

```diff
@@ -0,0 +1,304 @@
+			},
+			{
+				Name:        "scrape_java_script",
+				Type:        proto.ColumnType_BOOL,
+				Description: "Check if the project has scrape java script enabled.",
+			},
+			{
+				Name:        "scrub_ip_addresses",
+				Type:        proto.ColumnType_BOOL,
+				Description: "Check if the project has scrub IP addresses enabled.",
+				Transform:   transform.FromField("ScrubIPAddresses"),
+			},
+			{
+				Name:        "security_token",
+				Type:        proto.ColumnType_STRING,
+				Description: "The security token of the project.",
+			},
+			{
+				Name:        "security_token_header",
+				Type:        proto.ColumnType_STRING,
+				Description: "The security token header of the project.",
+			},
+			{
+				Name:        "subject_prefix",
+				Type:        proto.ColumnType_STRING,
+				Description: "The subject prefix of the project.",
+			},
+			{
+				Name:        "subject_template",
+				Type:        proto.ColumnType_STRING,
+				Description: "The subject template of the project.",
+			},
+			{
+				Name:        "verify_ssl",
+				Type:        proto.ColumnType_BOOL,
+				Description: "Check if the project is verified with SSL.",
+				Transform:   transform.FromField("VerifySSL"),
+			},
+			{
+				Name:        "allowed_domains",
+				Type:        proto.ColumnType_JSON,
+				Description: "The allowed domains of the project.",
+			},
+			{
+				Name:        "avatar",
+				Type:        proto.ColumnType_JSON,
+				Description: "Represents an avatar.",
+			},
+			{
+				Name:        "features",
+				Type:        proto.ColumnType_JSON,
+				Description: "The features of the project.",
+			},
+			{
+				Name:        "filters",
+				Type:        proto.ColumnType_JSON,
+				Description: "The project filters.",
+				Hydrate:     getProjectFilters,
+				Transform:   transform.FromValue(),
+			},
+			{
+				Name:        "options",
+				Type:        proto.ColumnType_JSON,
+				Description: "The project options.",
+			},
+			{
+				Name:        "organization",
+				Type:        proto.ColumnType_JSON,
+				Description: "The organization to which the project belongs to.",
+			},
+			{
+				Name:        "safe_fields",
+				Type:        proto.ColumnType_JSON,
+				Description: "The project safe fields.",
+			},
+			{
+				Name:        "sensitive_fields",
+				Type:        proto.ColumnType_JSON,
+				Description: "The project sensitive fields.",
+			},
+			{
+				Name:        "teams",
+				Type:        proto.ColumnType_JSON,
+				Description: "The project teams.",
+			},
+
+			/// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: "Title of the resource.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+		},
+	}
+}
+
+func listProjects(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	conn, err := getClient(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("listProjects", "connection_error", err)
```

Comment:
```suggestion
		plugin.Logger(ctx).Error("sentry_project.listProjects", "connection_error", err)
```
@bigdatasourav could you please append the table name to the error logs?

Reply:
updated